include/motor-control/core/stepper_motor/motion_controller.hpp

```diff
@@ -189,7 +189,8 @@ class MotionController {
         if (hardware.is_timer_interrupt_running()) {
             hardware.request_cancel();
         }
-        disable_motor();
+        hardware.deactivate_motor();
```

Comment:
do we need a delay here?

Reply:
no, this is about setting logical state. also it's already been tested and merged into `main`, this is a cherry-pick basically